app/controllers/api_controller.rb

```diff
@@ -58,32 +58,32 @@ def set_access_control_headers
   include_concern 'Authentication'
   include_concern 'AutomationRequests'
   include_concern 'Categories'
-  include_concern 'CustomAttributes'
   include_concern 'Conditions'
-  include_concern 'Groups'
-  include_concern 'Policies'
-  include_concern 'PolicyActions'
-  include_concern 'Providers'
+  include_concern 'CustomAttributes'
   include_concern 'Events'
   include_concern 'Features'
+  include_concern 'Groups'
   include_concern 'Hosts'
   include_concern 'Instances'
+  include_concern 'Policies'
+  include_concern 'PolicyActions'
+  include_concern 'Providers'
   include_concern 'ProvisionRequests'
   include_concern "Rates"
   include_concern "Reports"
   include_concern 'RequestTasks'
   include_concern 'ResourceActions'
   include_concern 'Roles'
-  include_concern 'Services'
   include_concern 'ServiceDialogs'
   include_concern 'ServiceOrders'
   include_concern 'ServiceRequests'
+  include_concern 'Services'
   include_concern 'ServiceTemplates'
   include_concern 'Settings'
   include_concern 'Software'
   include_concern 'Tags'
-  include_concern 'Tenants'
   include_concern 'TenantQuotas'
+  include_concern 'Tenants'
```

Comment:
This concerns me @abellotti @imtayadeway since module inclusion order is important.  I'd love to say this plethora of concerns could go in any order but I've looked at too many sporadic failures to believe that.